packages/sdk/test-next/messageStatus.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, expect, it } from 'vitest'
+
+import { CrossChainMessenger, MessageStatus } from '../src'
+import { l1Provider, l2Provider } from './testUtils/ethersProviders'
+
+const crossChainMessenger = new CrossChainMessenger({
+  l1SignerOrProvider: l1Provider,
+  l2SignerOrProvider: l2Provider,
+  l1ChainId: 5,
+  l2ChainId: 420,
+  bedrock: true,
+})
+
+describe('prove message', () => {
+  it(`should be able to correctly find a finalized withdrawal`, async () => {
```

Comment:
```suggestion
  it(`should be able to correctly identify a finalized withdrawal`, async () => {
```
test title is kinda weak